app/src/main/java/org/oppia/android/app/translation/AppLanguageWatcherMixin.kt

```diff
@@ -62,9 +66,20 @@ class AppLanguageWatcherMixin @Inject constructor(
     }
 
     // TODO(#52): Hook this up properly to profiles, and handle the non-profile activity cases.
```

Comment:
Seems like this can be removed now, right? It seems addressed in this PR.

Reply:
Yeah updated the same.